### PR TITLE
feat: add a support for arm64 image

### LINKED ISSUE #702: switcher_webapi docker image arm64
Could you please add a docker image for arm64 configuration for running it on PiOS 64bit variant?

### PR DESCRIPTION
**Pull request by YogevBokobza**
## Description

> Follow up issue https://github.com/TomerFi/switcher_webapi/issues/702 add the needed support.

**Related issue (if any):** fixes [#702](https://github.com/TomerFi/switcher_webapi/issues/702)

## Checklist

- [x] I have followed this repository's contributing guidelines.
- [x] I will adhere to the project's code of conduct.

## Additional information

> Anything else?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+services:
```
Comment (TomerFi) [inline docker-compose.yml]: What's the docker-compose file for?

**Review by TomerFi — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (YogevBokobza) [reply docker-compose.yml]: That's for local deployment.
Its building the docker image and runs it on a computer.
I saw Dima is using it (as mentioned in the other issue) so I said it's a good idea to have it here too.

Comment (TomerFi) [reply docker-compose.yml]: It's more of a user preference. We can add a section to the documentation site. We don't need it in the repo's root.

Comment (YogevBokobza) [reply docker-compose.yml]: From my experience, it's common to leave docker-compose file on the git as a file not as a doc.
Anyway, this is not harmful at all.

File: Makefile [hunk 2]
```diff
@@ -6,7 +6,7 @@ CURRENT_DATE = $(strip $(shell date -u +"%Y-%m-%dT%H:%M:%SZ"))
 CODE_VERSION = $(strip $(shell cat VERSION))
 FULL_IMAGE_NAME = $(strip $(IMAGE_NAME):$(CODE_VERSION))
 
-PLATFORMS = linux/amd64,linux/arm/v7
+PLATFORMS = linux/amd64,linux/arm/v7,linux/arm64/v8
```
Comment (TomerFi) [inline Makefile:9]: This should also be included in the following
- https://github.com/TomerFi/switcher_webapi/blob/dev/.github/workflows/pr.yml#L93
- https://github.com/TomerFi/switcher_webapi/blob/dev/.github/workflows/release.yml#L94
- https://github.com/TomerFi/switcher_webapi/blob/dev/.github/workflows/stage.yml#L66

File: docker-compose.yml (hunk 1 above)
Comment (TomerFi) [reply docker-compose.yml]: I think we can drop it. Feels unrelated.

Comment (TomerFi) [reply docker-compose.yml]: @YogevBokobza, let's remove the docker-compose file and merge. The arm64 is essential enough for a quick release.

**Review by TomerFi — APPROVED** [review]

**Pull request merged**